test/common/schemaform/review/ObjectField.unit.spec.jsx

```diff
@@ -123,7 +123,7 @@ describe('Schemaform review: ObjectField', () => {
     expect(tree.everySubTree('.form-review-panel-page-header-row')).not.to.be.empty;
     expect(tree.subTree('.form-review-panel-page-header').text()).to.be.empty;
   });
-  it('should hide expand under items when false', () => {
+  it('should hide expand under items when collapsed', () => {
```

Comment:
This seems like an odd change...Why?

Reply:
We used to hide expand under items when the value of the property being expanded under was false. Now, we test against the value of `expandUnderCondition` as well, with that logic living in the reducer code. The actual logic in this component is just to check against `ui:collapsed`.